components/Hero/Hero.js

```diff
@@ -0,0 +1,46 @@
+import Section from '../Section';
+import Decorator from '../Decorator';
+import DownloadButton from '../DownloadButton';
+import styles from './Hero.scss';
+
+const Hero = () => (
```

Comment:
Many of the circles have the exact same Y-animation, which looks a bit repetitive. They could move independently from each other? If that happens the animations would need to be a bit more subtle/slow as well or it may look chaotic. cc @fabric-8